common/src/main/java/tachyon/security/User.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Licensed to the University of California, Berkeley under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package tachyon.security;
+
+import java.security.Principal;
+
+public class User implements Principal {
+  private String mName;
+
+  // TODO: add more attributes and methods for supporting Kerberos
+
+  public User(String mName) {
+    this.mName = mName;
+  }
+
+  @Override
+  public String getName() {
+    return mName;
+  }
+
+  @Override
+  public boolean equals(Object o) {
+    if (this == o) {
+      return true;
```

Comment:
Will this ever be true and `mName.equals(((User) o).mName)` false? If not, perhaps its clearer to skip this condition?